indico/modules/events/registration/fields/base.py

```diff
@@ -137,7 +137,8 @@ def unprocess_field_data(cls, versioned_data, unversioned_data):
 
     @property
     def view_data(self):
-        return self.unprocess_field_data(self.form_item.versioned_data, self.form_item.data)
+        return (self.unprocess_field_data(self.form_item.versioned_data, self.form_item.data) |
+                {'defaultValue': self.default_value})
```

Comment:
I think we camelize the data later so we could still use snake case here, since that's what `unprocess_field_data` returns as well.